README.md

```diff
@@ -1,16 +1,49 @@
 # React Design Patterns
 
```

Comment:
Can we add a CircleCI badge here (maybe also log an enhancement to break the CircleCI tests into separate workflows and we can have a separate badge for each pattern perhaps?).